content/enterprise/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Enterprise solutions in the Dapr ecosystem"
+#subtitle: ""
+# meta description
+description: "Enterprise solutions for the Dapr ecosystem"
+draft: false
+
+companies:
+  - logo : "images/enterprise/azure.png"
+    alt: "Azure Container Apps"
+    title: "Azure Container Apps"
+    summary: "Azure Container Apps is a fully managed Kubernetes-based application platform that helps you deploy apps from code or containers without orchestrating complex infrastructure. Build heterogeneous modern apps or microservices with unified centralized networking, observability, dynamic scaling, and configuration for higher productivity. Design resilient microservices with full support for Dapr and dynamic scaling powered by KEDA."
+    cta :
+      enable : true
+      label : "Visit website"
+      link : "https://azure.microsoft.com/en-us/products/container-apps/"
+
+  - logo : "images/enterprise/diagrid.png"
+    alt: "Diagrid Conductor"
+    title: "Diagrid Conductor"
+    summary: "Let Conductor manage Dapr on your Kubernetes cluster so you can focus on writing code. Turn your Kubernetes cluster into a microservices powerhouse. Drastically reduce Dapr operational overhead while increasing reliability and security. Gain unparalleled visibility into your applications and infrastructure using the telemetry data generated by Dapr. Conductor tracks the health of your entire system and immediately shows you when something is wrong with built-in alerting for every Dapr metric."
```

Comment:
Change to: `Fully automated Dapr on any Kubernetes cluster with open-source enterprise support. Drastically reduce Dapr operational overhead while increasing reliability and security. Gain unparalleled visibility into your applications and infrastructure using the telemetry data generated by Dapr. Conductor tracks the health of your entire system and immediately shows you when something is wrong with built-in alerting for every Dapr metric.`